qiskit_experiments/library/tomography/tomography_experiment.py

```diff
@@ -177,9 +211,9 @@ def circuits(self):
             if prep_element:
                 # Add tomography preparation
                 prep_circ = self._prep_circ_basis.circuit(prep_element, self._prep_physical_qubits)
-                circ.reset(self._prep_qubits)
-                circ.compose(prep_circ, self._prep_qubits, inplace=True)
-                circ.barrier(self._prep_qubits)
+                circ.reset(self._prep_indices)
```

Comment:
Maybe I'm just confused, but I was expecting this to use `_prep_physical_qubits`.

Same comment for measurements.

Reply:
This is constructing the "logical" N-qubit circuit, so uses the indices. It then later gets transpiled to the physical qubits according to base experiments regular execution. Hence why the old variable name was a bit confusing.